### PR TITLE
Fix internal linking on config-settings.rst 

### PR DESCRIPTION
**Pull request by esethna**

### CODE REVIEW

**Comment by it33** [comment]
Omg, so much nicer.

@esethna maybe as LI for help [testing links](https://github.com/mattermost/docs/blob/810474fb68c81843d615c17d754a98fab4321d5a/source/administration/config-settings.rst#localization) and we can get this merged?

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -69,8 +69,10 @@ gzip compression applies to the HTML, CSS, Javascript, and other static content
 | This feature's ``config.json`` setting is ``"WebserverMode": "gzip"`` with options ``gzip``, ``uncompressed`` and ``disabled`` for above settings respectively.      |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
-Reload Configuration from Disk (Enterprise)
+Reload Configuration from Disk
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ 
+*Available in Enterprise Edition E10 and higher*
```
Comment (it33) [inline source/administration/config-settings.rst]: Should be E20, this feature is for HA

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -516,8 +524,10 @@ Enter ``https://<your-gitlab-url>/oauth/authorize`` (example: ``https://example.
 
 ________
 
-Google (Enterprise)
+Google
 ``````````````````````````
+*Available in Enterprise Edition E10 and higher*
```
Comment (it33) [inline source/administration/config-settings.rst]: Should be E20

File: source/administration/config-settings.rst [hunk 3]
```diff
@@ -570,8 +580,10 @@ It is recommended to use `https://www.googleapis.com/oauth2/v4/token` as the Tok
 
 ________
 
-Office 365 (Enterprise)
+Office 365
 ``````````````````````````
+*Available in Enterprise Edition E10 and higher*
```
Comment (it33) [inline source/administration/config-settings.rst]: Should be E20

File: source/administration/config-settings.rst [hunk 4]
```diff
@@ -1797,8 +1828,10 @@ Trace
 | This feature's ``config.json`` setting is ``"Trace": false`` with options ``true`` and ``false`` for above settings respectively.                                    |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
-Recycle Database Connections (Enterprise)
+Recycle Database Connections
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+*Available in Enterprise Edition E10 and higher*
```
Comment (it33) [inline source/administration/config-settings.rst]: E20, this is for HA

**Comment by shieldsjared** [comment]
+1

**Comment by esethna** [comment]
Thanks, updated!

**Pull request merged**